Assets/Scripts/ManagerScripts/RespawnManager.cs

```diff
@@ -52,13 +52,19 @@ public IEnumerator RespawnPlayer(float respawnClipLength)
         AdjustMusicTime();
         //AdjustMidiTime();
         yield return new WaitForSeconds(respawnClipLength - 5f);
+        // StartCoroutine(RespawnPlayerAfterCountdown());
         AdjustPlayerPosition();
+        _playerCharacterMovement.enabled = false;
+        PlayerMovement.Current.walkingSound.Stop();
+        CountdownManager.Current.SetCountdown(3f);
+        yield return new WaitForSeconds(3f);
         scoreManager.SetAndUpdateFishScore(_playerFishScore);
```

Comment:
Hmm it's kinda weird how you respawn, then it countsdown, and then your score goes away lol. I think the countdown should be right before " _playerCharacterMovement.enabled = true;"

Reply:
Yeah I tested this and it seems to work much better:
![image](https://user-images.githubusercontent.com/45300621/204702626-52dbb510-eacf-4cbd-957c-e3403cf03c5e.png)

Reply:
Also can you make ScoreManager a singleton lol Idk why it's an attribute (referring to lines 59 and 60 in the pic from the last comment ^)